packages/env/lib/config.js

```diff
@@ -150,11 +150,10 @@ module.exports = {
 			);
 		}
 
-		const workDirectoryPath = path.resolve(
-			os.homedir(),
-			'.wp-env',
-			md5( configPath )
-		);
+		const wpEnvHome =
+			process.env.WP_ENV_SOURCE || `${ os.homedir() }/.wp-env`;
```

Comment:
It's probably not needed, but I'd sleep better if we use `path.join` instead of hardcoding a path separator in a template literal like that.

Reply:
Yes we'll need to use `path.join()` so that this works on Windows.

Reply:
Good call.